README.md

```diff
@@ -23,8 +23,191 @@ Terra Toolkit is a utility module used to facilitate independent development of
 
 - Install with [npm](https://www.npmjs.com): `npm install terra-toolkit --save-dev`
 
+## Webdriver.io
+[Webdriver.io](http://webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. Webdriver.io also provides easy services for setting up selenium, starting webpack and static servers, as well as accessibilty and visual regression testing.
 
-## Nightwatch Utility
+
+Install the package
+```sh
+npm install webdriverio --save-dev
+```
+Generate a configuration file. [See documentation](http://webdriver.io/guide/testrunner/gettingstarted.html) for more info.
```

Comment:
I prefer that we provide a "recommended" configuration file.

Reply:
Recommended for what setup? Jenkins runs, Travis, local, hitting our internal selenium?

Reply:
Yes for all of those cases.  I'd like to provide some recommendations to the 9 questions that get asked by running "./node_modules/.bin/wdio config".

Reply:
So I guess not *one* recommended file, but recommendations for the common scenarios our consumers will be dealing with.